<inh f='include/c++/12/bits/exception.h' l='61' c='std::exception'/>
<def f='include/c++/12/variant' l='1304' ll='1319'/>
<use f='include/c++/12/variant' l='1324' c='_ZSt26__throw_bad_variant_accessPKc'/>
<size>16</size>
<fun r='_ZNSt18bad_variant_accessC1Ev'/>
<fun r='_ZNKSt18bad_variant_access4whatEv'/>
<fun r='_ZNSt18bad_variant_accessC1EPKc'/>
<mbr r='std::bad_variant_access::_M_reason' o='64' t='const char *'/>
